configure.ac

```diff
@@ -13,6 +13,9 @@ AC_DEFINE(VMA_LIBRARY_MINOR, vma_ver_minor, [VMA Minor Version])
 AC_DEFINE(VMA_LIBRARY_REVISION, vma_ver_revision, [VMA Revision])
 AC_DEFINE(VMA_LIBRARY_RELEASE, vma_ver_release, [VMA Release])
 
+define([vma_git_version], esyscmd([sh -c "touch ./build/current-version; head -1 ./build/current-version  | tr -d '\n'"]))
```

Comment:
current-version file is temporarily created and immediately deleted
versioning.sh file was intended to be part of VMA tree (so none of the files is a leftover auxiliary).
Nevertheless your suggestion for calculating "on the fly" without additional files was accepted